apps/web/src/app/vault/individual-vault/vault-filter/components/vault-filter.component.ts

```diff
@@ -105,24 +111,49 @@ export class VaultFilterComponent implements OnInit, OnDestroy {
   }
 
   applyOrganizationFilter = async (orgNode: TreeNode<OrganizationFilter>): Promise<void> => {
-    if (!orgNode?.node.enabled) {
+    if (!orgNode?.node?.enabled) {
       this.platformUtilsService.showToast(
         "error",
         null,
         this.i18nService.t("disabledOrganizationFilterError"),
       );
+      const sub = await this.organizationApiService.getSubscription(orgNode?.node?.id);
+      if (sub && sub?.subscription?.status === "unpaid") {
```

Comment:
No need for sub here

```suggestion
      if (sub?.subscription?.status === "unpaid") {
```